id/news/_posts/2019-03-31-support-of-ruby-2-3-has-ended.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: news_post
+title: "Dukungan Ruby 2.3 telah berakhir"
+author: "antonpaisov"
+translator: "meisyal"
+date: 2019-03-31 00:00:00 +0000
+lang: id
+---
+
+Kami mengumumkan semua dukungan dari rangkaian Ruby 2.3 telah berakhir.
+
+Setelah rilis Ruby 2.3.7 pada 28 Maret 2018,
+dukungan dari rangkaian Ruby 2.3 berada pada fase perawatan keamanan.
+Sekarang, setelah satu tahun berlalu, fase tersebut telah berakhir.
+Sehingga, pada 31 Maret 2019, semua dukungan dari rangkaian Ruby 2.3 berakhir.
+Perbaikan keamanan dan *bug* dari versi Ruby terbaru tidak akan
+di-*backport* ke 2.3. Tidak akan ada juga *patch* untuk 2.3.
+Kami sangat merekomendasikan Anda untuk memperbarui Ruby ke 2.6 atau 2.5
+segera mungkin.
```

Comment:
better to use "sesegera mungkin"

Reply:
updated to "sesegera mungkin", @kuntoaji